### PR TITLE
Remove filtered displaySets from parameter from study protocolEngine.findMatch()

### PR DESCRIPTION
**Pull request by Sofien-Sellami**
### Context
This pull request removes the studyDisplaySets parameter from the protocolEngine.findMatch() call in order to access a list of all expanded displaySets in studyMatchDetails.

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [x] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [x] "OS: 
- [x] "Node version: 
- [x] "Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/core/src/services/HangingProtocolService/HangingProtocolService.ts [hunk 1]
```diff
@@ -1303,15 +1303,13 @@ export default class HangingProtocolService extends PubSubService {
     this.studies.forEach(study => {
       // Skip non-active if active only
       if (matchActiveOnly && this.activeStudy !== study) return;
-      const studyDisplaySets = this.displaySets.filter(
-        it => it.StudyInstanceUID === study.StudyInstanceUID
-      );
+
```
[the remark this answers is not in the document]
Comment (wayfarer3130) [reply platform/core/src/services/HangingProtocolService/HangingProtocolService.ts:1392]: Thinking it over, probably leaving displaySets as is, but passing allDisplaySets into the findMatch function would be more consistent.  The rule is testing this study right now, so it should have the display sets for this study, not all display sets, and if a particular rule wants to look at all display sets, then just having it as a another value would make that possible.

**Review by wayfarer3130 — CHANGES_REQUESTED** [review]
Can you add a hanging protcol example and test set that requires this change to the basic test mode?  
Also, I'd like to see this as an addition to the findMatch rules as opposed to a change to what is currently passed in.  There are use cases this is needed, but having the display sets for the study is used in several existing hanging protocols, which would break them when being used with multi-study display sets.

**Comment by salimkanoun** [comment]
The change only exposes the displayset at the study matching, at series level display set are only studies display sets. Did the study matching worked some days ? What we understood is that study matching as it was not passing displays set belonging to other studies was probably unable to match any other study outside the current one. (and neither pass other known studies as this.studies in HPservice is now updated to recieve new known studies)

**Comment by wayfarer3130** [comment]
Wait till the PR checks pass, then I can merge.

**Comment by salimkanoun** [comment]
Dear @wayfarer3130 should be good this one too ?

**Pull request merged**